src/main/java/ChatGPT.java

```diff
@@ -0,0 +1,234 @@
+                    System.out.println("\t" + LINE);
+                    System.out.println(list.get(index-1).setCompleted(false));
+                    System.out.println("\t" + LINE);
+                    break;
+
+                case DELETE:
+                    index = userInput.nextInt();
+                    Task deletedTask = list.get(index-1);
+                    list.remove(index-1);
+                    sendDeleteMsg(deletedTask);
+                    break;
+                }
+            } catch (IllegalArgumentException | NoSuchMethodException e) {
+                System.out.println("\t" + LINE);
+                System.out.println(e.getMessage());
+                System.out.println("\t" + LINE);
+
+            } catch (Exception e) {
+                System.out.println("\t" + LINE);
+                System.out.println("\t There seems to have been a problem with your inputs Q.Q" +
+                        "\n\t Please try again with a valid command and inputs");
+                System.out.println("\t" + LINE);
+            }
+            saveData();
+        } while (true);
+    }
+
+    private static void sendGreeting() {
+        System.out.println("\t"+LINE);
+        System.out.println("\tHello! I'm " + NAME);
+        System.out.println("\tWhat can I do for you?");
+        System.out.println("\t"+LINE);
+    }
+
+    private static void sendExit() {
+        System.out.println("\t"+LINE);
+        System.out.println("\tBye. Hope to see you again soon!");
+        System.out.println("\t"+LINE);
+    }
+
+    private static void sendAddTaskMsg(Task task) {
+        System.out.println("\t"+LINE);
+        System.out.println("\tGot it. I've added this task:");
+        System.out.println("\t  "+ task.toString());
+        System.out.println("\tNow you have " + list.size() + " tasks in your list.");
+        System.out.println("\t"+LINE);
+    }
+
+    private static void sendDeleteMsg(Task task) {
+        System.out.println("\t"+LINE);
+        System.out.println("\tNoted. I've removed this task:");
+        System.out.println("\t  "+ task.toString());
+        System.out.println("\tNow you have " + list.size() + " tasks in your list.");
+        System.out.println("\t"+LINE);
+    }
+
+    private static void printList() {
+        System.out.println("\t"+LINE);
+        if (list.isEmpty()) {
+            System.out.println("\tNothing has been added");
+        } else {
+            System.out.println("\tHere are the tasks in your list:");
+        }
+        for (int i = 0; i < list.size(); i++){
+            System.out.println("\t" + (i+1) + ". " + list.get(i).toString());
+        }
+        System.out.println("\t"+LINE);
+    }
+
+    private static ArrayList<Task> readData() {
+        File currentDir = new File(".");
+        File dataDir = new File (currentDir.getAbsolutePath() + "\\data");
+        File data = new File (dataDir.getAbsolutePath() + "\\data.txt");
+        ArrayList<Task> result = new ArrayList<>();
```

Comment:
Perhaps plural form "results" can be used to represent this collection of tasks?